packages/js/components/src/experimental-select-control/menu.scss

```diff
@@ -1,17 +1,19 @@
 .woocommerce-experimental-select-control__menu {
-    position: absolute;
-    width: 100%;
-    top: 100%;
-    box-sizing: border-box;
-    display: none;
+	position: absolute;
+	width: 100%;
+	top: 100%;
+	box-sizing: border-box;
+	display: none;
+	z-index: 10;
+	margin-top: $gap-smaller;
```

Comment:
Mostly spacing it seems from spaces to tabs, this is what my `prettier` did. The only addition is:
```
z-index: 10;
margin-top: $gap-smaller;
```